aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -138,11 +138,11 @@ public Orchestrator(
         this.targetState = targetState;
         this.disposables = new CompositeDisposable();
 
-        // Operation times out after 10 seconds. If there are more than 5 models,
-        // then 2 seconds are added to the timer per additional model count.
+        // Operation times out after 60 seconds. If there are more than 5 models,
+        // then 20 seconds are added to the timer per additional model count.
         this.adjustedTimeoutSeconds = Math.max(
             NETWORK_OP_TIMEOUT_SECONDS,
-            TIMEOUT_SECONDS_PER_MODEL * modelProvider.models().size()
+            TIMEOUT_SECONDS_PER_MODEL * modelProvider.modelSchemas().size()
```

Comment:
Flutter uses the schemas, but native uses the models, right? Do you want to take the max of the two?
```suggestion
            TIMEOUT_SECONDS_PER_MODEL * Math.max(modelProvider.modelSchemas().size(), modelProvider.models().size())
```